content/questions/actions/question-021.md

```diff
@@ -5,7 +5,7 @@ title: "Question 021"
 draft: false
 ---
 
-> https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs
+> https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#inputs-context
```

Comment:
This questions specifically asks about "manually triggered workflows" so I believe the original link is better because that's exactly what `workflow_dispatch` is.

Also the link you provided does not say anything about `environment` input type sadly. I'd leave the original link tbh

I still think the first link is better, since it points to a page with an example that uses these i

Reply:
The original link doesn't contain the answer to the question. Please provide the specific text with the answer if you see that in the original link

Reply:
> This questions specifically asks about "manually triggered workflows" so I believe the original link is better because that's exactly what workflow_dispatch is.

It literally says that workflow_dispatch and reusable workflows have the same inputs. 
![image](https://github.com/user-attachments/assets/fb8a5aba-77c2-4141-87bf-5558362957a9)

Reply:
@FidelusAleksander Fixed. Please check

Reply:
Looks like the previous PR failed some checks? 
![image](https://github.com/user-attachments/assets/798821bb-1bde-49a2-974b-6daa43cf6dc8)